src/services/link/index.js

```diff
@@ -16,25 +16,49 @@ module.exports = function setup() {
 		lean: true,
 	};
 
+	const registrationLinkExpirationDays = app.get('registrationLinkExpirationDays') || 30;
+
 	let linkService = service(options);
 
-	function redirectToTarget(req, res, next) {
-		if (req.method === 'GET' && !req.query.target) { // capture these requests and issue a redirect
+	const hasExpired = (date) => {
+		// calculate the date the link will expire on
+		const expirationDate = new Date(date);
+		expirationDate.setDate(expirationDate.getDate() + registrationLinkExpirationDays);
+		// check if that expiration date has already passed
+		return expirationDate <= Date.now();
+	};
+
+	const isLocalRegistrationLink = (url) => {
+		// calculate the beginning of a registrationLink e.g. https://schul-cloud.org/registration/
+		const linkPrefix = `${app.settings.services.web}/registration/`;
```

Comment:
`/registration/` must not be part of the check, take care of the protocol for this check

Reply:
I don't understand. Could you explain?